operator/cmd/metrics/adapter/provider/custom_metrics.go

```diff
@@ -0,0 +1,106 @@
+package provider
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/metrics/pkg/apis/custom_metrics"
+	"sigs.k8s.io/custom-metrics-apiserver/pkg/provider"
+)
+
+var ErrMetricNotFound = errors.New("no metric value found")
```

Comment:
woudl it make sense to introduce a common.go in cmd/metrics/adapter folder and move all the variables, constants, errors there ?